components/progress/index.en-US.md

```diff
@@ -24,6 +24,7 @@ Properties that shared by all types.
 | percent | to set the completion percentage | number | 0 |  |
 | showInfo | whether to display the progress value and the status icon | boolean | true |  |
 | status | to set the status of the Progress, options: `success` `exception` `normal` `active`(line only) | string | - |  |
+| orientation | to set the orientation from horizontal to vertical, options: `default` `vertical` | string | - |  |
```

Comment:
It's better to set `horizontal` as a default value, for `default` may make people confused.

Reply:
done

Reply:
1. Maybe you forget to update this in docs. `default` => `horizontal`
2. The penultimate grid can be filled in with default values, like 
| options: `horizontal` `vertical` | string | `horizontal` |  3.24.0   |